docs/api.md

```
@@ -1329,7 +1473,11 @@ from docx_editor.exceptions import WorkspaceExistsError
 
 Raised when the workspace is out of sync with the source document: the source changed on disk since the workspace was created, or the workspace holds unsaved changes from a previous session that the source never received. Structured fields: `workspace_path` and `source_path`.
 
-`Document.open(path, force_recreate=True)` recovers but discards the workspace's unsaved edits. To rescue them first, save the orphaned workspace to a new file (`Workspace` is not exported at the package root — use the deep import):
+`Document.open(path, force_recreate=True)` and
+[`Document.discard_workspace(path)`](#documentdiscard_workspacepath-workspace_dirnone)
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)